installation/resources/crds/service-catalog/servicebindingusages.servicecatalog.crd.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
if we follow by resources rules then `servicebindingusages` should be in `service-catalog-addons`
see [resourses](https://github.com/kyma-project/kyma/tree/main/resources/service-catalog-addons/charts).
if there are no rules or rules ar different please let me know